genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -143,6 +143,18 @@ to work.
 |The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. This is property is ignored unless `genie.jobs.users.activeLimit.enabled` is set to true.
 |100
 
+|genie.jobs.completionCheckBackOff.minInterval
```

Comment:
I think you need to change these for Genie 4 as it builds on Boot 2 which changed how property binding works to enforce certain naming schemes. See [here](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.0-Migration-Guide#relaxed-binding). The gist is that non-leaf nodes of the name/path can't be camel case they need to use `-` or something.

Reply:
Their documentation is a little confusing (in so much as they say you need to use kebab case in the `prefix` of `@ConfigurationProperties` but say that camel case will work other places. I had some trouble during the Boot 2 migration with some of our existing properties and I had to switch them. Maybe they've changed something since then but better safe than sorry.

Reply:
From table 24.1 here: https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/reference/htmlsingle/#boot-features-external-config-relaxed-binding it seems like the current should still work, just not recommended.
We do have *a lot* of properties using camel.
I'll switch these to kebab.